rockr/views.py

```diff
@@ -91,11 +91,14 @@ def logout():
     return "success", 200
 
 
-@app.route("/change_password", methods=["POST"])
+@app.route("/reset_password", methods=["GET"])
```

Comment:
feels weird to have this be a GET request

Reply:
You know what Quaill.... you're right, changed to POST